homeassistant/components/aurora/coordinator.py

```diff
@@ -34,10 +34,12 @@ def __init__(
             update_interval=timedelta(minutes=5),
         )
 
-        self.api = api
-        self.latitude = int(latitude)
-        self.longitude = int(longitude)
-        self.threshold = int(threshold)
+        self.api = AuroraForecast(async_get_clientsession(hass))
+        self.latitude = int(self.config_entry.data[CONF_LATITUDE])
```

Comment:
why is the data in the config entry data not in the right type?

Reply:
That was also something i was wondering and wanted to add a migration for that